web/packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx

```diff
@@ -71,8 +67,13 @@ export function ConnectionItem(props: ConnectionItemProps) {
       onClick={props.onActivate}
       isActive={isActive}
       ref={ref}
+      $showClusterName={props.showClusterName}
       css={`
-        padding: 6px 8px;
+        padding: ${props => props.theme.space[1]}px
+          ${props => props.theme.space[2]}px;
+        // Space out items more if there are two lines of text to show inside a single item.
+        margin-block-start: ${props =>
```

Comment:
This is similar to `gap` but is defined on a child rather than a parent.

Well, it is similar to gap when called like this:

```css
.stack > * + * {
  margin-block-start: 1.5rem;
}
```

…so that it's not applied to the first item in the list, but here it doesn't matter anyway.

Reply:
I think using `display: flex` on `<ul>` shouldn't break anything, but I'm fine with keeping it as is.